exercises/practice/hello-world/hello-world.js

```diff
@@ -3,6 +3,4 @@
 // convenience to get you started writing code faster.
 //
 
-export const hello = () => {
-  throw new Error('Remove this statement and implement this function');
-};
+export const hello = () => 'Goodbye, Mars!';
```

Comment:
```suggestion
export function hello() {
  return 'Goodbye, Mars!';
}
```

I personally like this better, despite the `return`. What do you think @ErikSchierboom ?

Reply:
I definitely like that better. I just used whatever was in the example file.